packages/slate-react/src/utils/scroll-to-selection.js

```diff
@@ -38,11 +37,64 @@ function findScrollContainer(el) {
     parent = parent.parentNode
   }
 
-  if (!scroller) return window
```

Comment:
You have to explicitly set html and body to `overflow: visible /* or none */` in order for this to be hit. In the browsers I tested, the default value for both is `auto`.